frontend/public/components/namespace.jsx

```diff
@@ -567,138 +567,124 @@ const projectRowStateToProps = ({ UI }) => ({
 });
 
 const ProjectTableRow = connect(projectRowStateToProps)(
-  withTranslation()(
-    withUserSettingsCompatibility(
+  withTranslation()(({ obj: project, index, rowKey, style, customData = {}, metrics, t }) => {
+    const [tableColumns, , loaded] = useUserSettingsCompatibility(
       COLUMN_MANAGEMENT_CONFIGMAP_KEY,
       COLUMN_MANAGEMENT_LOCAL_STORAGE_KEY,
-      undefined,
+      null,
```

Comment:
Please keep the default as undefined so that this value are not stored in user settings configmap until the user changes the table config.